lib/sidekiq/cron/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Cron
+    VERSION = "1.2.0"
```

Comment:
shall we bump it ?

Reply:
I'm happy to bump it if that's the desire of the maintainers.  I did limit Ruby version to 2.6+, so that may be worth a version bump.

Reply:
1.2.0 is already release and `master` branch already have mentions a 1.3.0 in progress.
I think 1.3.0 would be good.
hopefully someone will release into rubygems

Reply:
https://github.com/ondrejbartas/sidekiq-cron/commit/409f11e6eae47df59adcd13cdac4f3c7b5157647